Server/JokesOnYou.Web.Api/Controllers/JokesController.cs

```diff
@@ -54,7 +54,7 @@ public async Task<ActionResult<JokeReplyDto>> UpdateJoke(JokeUpdateDto jokeUpdat
 
         [AllowAnonymous]
         [HttpGet("{id}")]
-        public async Task<ActionResult<JokeReplyDto>> GetJoke(int id)
+        public async Task<ActionResult<JokeWithAuthorReplyDto>> GetJoke(int id)
```

Comment:
I think this should just be JokeDto, we can reuse this model as we always want the author. Naming it both reply and dto is not needed (yes i know we have it on some places) but we have an issue for renaming theese to just responses instead

Reply:
The idea of the issue was to only include AuthorName when you retrieve a specific Joke. For example when retrieving a list of Jokes, maybe you don't want to include the Authors (could potentially be a lot of different authors).

With that in mind, do you still think it should be the same Dto? If yes, will we just leave AuthorName empty when retrieving a list of Jokes or also populate the AuthorName in that case?

Reply:
always populate i suppose, since it does make sense to always have the author username presented to the user